packages/paste-website/src/pages/components/button/api.mdx

```diff
@@ -0,0 +1,59 @@
+export const meta = {
+  title: 'Button - API',
+  package: '@twilio-paste/button',
+  description: 'A Button communicates that users can trigger an action.',
```

Comment:
Did you intentionally change the package description? It's currently:

> A Button is a clickable element which communicates that users can trigger an action.

Which better follows the format of the rest of our package descriptions

Reply:
No changes, it's just copy and pasted from the existing pages.